src/Symfony/Component/Console/Input/StringInput.php

```diff
@@ -52,12 +52,12 @@ private function tokenize($input)
         $length = \strlen($input);
         $cursor = 0;
         while ($cursor < $length) {
-            if (preg_match('/\s+/A', $input, $match, null, $cursor)) {
-            } elseif (preg_match('/([^="\'\s]+?)(=?)('.self::REGEX_QUOTED_STRING.'+)/A', $input, $match, null, $cursor)) {
+            if (preg_match('/\s+/A', $input, $match, 0, $cursor)) {
+            } elseif (preg_match('/([^="\'\s]+?)(=?)('.self::REGEX_QUOTED_STRING.'+)/A', $input, $match, 0, $cursor)) {
                 $tokens[] = $match[1].$match[2].stripcslashes(str_replace(['"\'', '\'"', '\'\'', '""'], '', substr($match[3], 1, \strlen($match[3]) - 2)));
-            } elseif (preg_match('/'.self::REGEX_QUOTED_STRING.'/A', $input, $match, null, $cursor)) {
+            } elseif (preg_match('/'.self::REGEX_QUOTED_STRING.'/A', $input, $match, 0, $cursor)) {
                 $tokens[] = stripcslashes(substr($match[0], 1, \strlen($match[0]) - 2));
-            } elseif (preg_match('/'.self::REGEX_STRING.'/A', $input, $match, null, $cursor)) {
+            } elseif (preg_match('/'.self::REGEX_STRING.'/A', $input, $match, 0, $cursor)) {
```

Comment:
Suggestion only:  The last two parameters are optional (flags and offset) as we only set them to the natural default value would removing these parameters be a better way?

Reply:
Nope, `$cursor` can be different than `0` since line 67:
https://github.com/symfony/symfony/pull/40187/files#diff-d75b469ce202adac8ab47198ebc465f6d34c7b9f7aaf2c52835a502a278b38deR67